pom.xml

```diff
@@ -1835,6 +1835,8 @@
                                 <exclude>.editorconfig</exclude>
                                 <exclude>**/hadoop.indexer.libs.version</exclude>
                                 <exclude>**/codegen/**</exclude>
+                                <exclude>website/target/**</exclude>
```

Comment:
This looks like a setup issue to me. 
Please remove these excludes.

Reply:
@cryptoe done.